src/base/bittorrent/torrentinfo.cpp

```diff
@@ -246,6 +246,28 @@ QVector<int> TorrentInfo::fileIndicesForPiece(int pieceIndex) const
     return res;
 }
 
+QByteArray TorrentInfo::hashForPiece(const int pieceIndex) const
+{
+    if (!isValid() || (pieceIndex < 0) || (pieceIndex >= piecesCount()))
+        return {};
+    return { m_nativeInfo->hash_for_piece_ptr(pieceIndex), libtorrent::sha1_hash::size };
+}
+
+QVector<QByteArray> TorrentInfo::piecesHashes() const
```

Comment:
I'm not a big expert in English, but as I recall it is incorrect to use the combination of the two words in the plural. `pieceHashes` is correct. @LordNyriox?